[2023-12-28T16:09:06.978+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:09:06.980+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:09:07.195+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:09:07.199+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:09:07.374+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-19T04:00:00+00:00 [queued]>
[2023-12-28T16:09:07.410+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-19T04:00:00+00:00 [queued]>
[2023-12-28T16:09:07.411+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-28T16:09:07.488+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Create_Table> on 2023-12-19 04:00:00+00:00
[2023-12-28T16:09:07.497+0000] {standard_task_runner.py:57} INFO - Started process 558 to run task
[2023-12-28T16:09:07.506+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Create_Table', 'scheduled__2023-12-19T04:00:00+00:00', '--job-id', '768', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmpxi6bynvg']
[2023-12-28T16:09:07.509+0000] {standard_task_runner.py:85} INFO - Job 768: Subtask Create_Table
[2023-12-28T16:09:07.651+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:09:07.653+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:09:07.672+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Create_Table scheduled__2023-12-19T04:00:00+00:00 [running]> on host d67dd4682ccf
[2023-12-28T16:09:08.036+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-19T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-19T04:00:00+00:00'
[2023-12-28T16:09:08.067+0000] {base.py:73} INFO - Using connection ID 'neonServerDB-Postgres' for task execution.
[2023-12-28T16:09:12.822+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/airQuality.py", line 133, in createTableSQL
    cursor.execute(createQuery)
psycopg2.errors.UniqueViolation: duplicate key value violates unique constraint "pg_class_relname_nsp_index"
DETAIL:  Key (relname, relnamespace)=(air_quality_id_seq, 2200) already exists.

[2023-12-28T16:09:12.854+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=air_quality, task_id=Create_Table, execution_date=20231219T040000, start_date=20231228T160907, end_date=20231228T160912
[2023-12-28T16:09:12.889+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 768 for task Create_Table (duplicate key value violates unique constraint "pg_class_relname_nsp_index"
DETAIL:  Key (relname, relnamespace)=(air_quality_id_seq, 2200) already exists.
; 558)
[2023-12-28T16:09:12.913+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-12-28T16:09:12.977+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-12-28T16:52:11.275+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:52:11.282+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:52:11.392+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:52:11.393+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:52:11.542+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-19T04:00:00+00:00 [queued]>
[2023-12-28T16:52:11.587+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-19T04:00:00+00:00 [queued]>
[2023-12-28T16:52:11.592+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-28T16:52:11.688+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Create_Table> on 2023-12-19 04:00:00+00:00
[2023-12-28T16:52:11.697+0000] {standard_task_runner.py:57} INFO - Started process 354 to run task
[2023-12-28T16:52:11.736+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Create_Table', 'scheduled__2023-12-19T04:00:00+00:00', '--job-id', '917', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmpgxzof3hu']
[2023-12-28T16:52:11.753+0000] {standard_task_runner.py:85} INFO - Job 917: Subtask Create_Table
[2023-12-28T16:52:12.087+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:52:12.089+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:52:12.093+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Create_Table scheduled__2023-12-19T04:00:00+00:00 [running]> on host d67dd4682ccf
[2023-12-28T16:52:12.634+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-19T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-19T04:00:00+00:00'
[2023-12-28T16:52:12.703+0000] {base.py:73} INFO - Using connection ID 'neonServerDB-Postgres' for task execution.
[2023-12-28T16:52:13.317+0000] {airQuality.py:139} INFO - Table air_quality Already Exist
[2023-12-28T16:52:13.373+0000] {python.py:194} INFO - Done. Returned value was: 
        CREATE TABLE air_quality (
            id serial primary key, 
            city varchar(250),
            country varchar(250), 
            unit varchar(20), 
            last_value float8, 
            lat float8,
            lng float8
        )
    
[2023-12-28T16:52:13.495+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=air_quality, task_id=Create_Table, execution_date=20231219T040000, start_date=20231228T165211, end_date=20231228T165213
[2023-12-28T16:52:13.633+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-28T16:52:13.778+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-12-28T16:55:11.812+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:55:11.815+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:55:11.940+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:55:11.942+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:55:12.169+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-19T04:00:00+00:00 [queued]>
[2023-12-28T16:55:12.237+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-19T04:00:00+00:00 [queued]>
[2023-12-28T16:55:12.246+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-28T16:55:12.391+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Create_Table> on 2023-12-19 04:00:00+00:00
[2023-12-28T16:55:12.404+0000] {standard_task_runner.py:57} INFO - Started process 1321 to run task
[2023-12-28T16:55:12.418+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Create_Table', 'scheduled__2023-12-19T04:00:00+00:00', '--job-id', '1154', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmpprq08u85']
[2023-12-28T16:55:12.425+0000] {standard_task_runner.py:85} INFO - Job 1154: Subtask Create_Table
[2023-12-28T16:55:12.632+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:55:12.640+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-19T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:55:12.646+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Create_Table scheduled__2023-12-19T04:00:00+00:00 [running]> on host d67dd4682ccf
[2023-12-28T16:55:12.969+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-19T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-19T04:00:00+00:00'
[2023-12-28T16:55:12.990+0000] {base.py:73} INFO - Using connection ID 'neonServerDB-Postgres' for task execution.
[2023-12-28T16:55:13.496+0000] {airQuality.py:139} INFO - Table air_quality Already Exist
[2023-12-28T16:55:13.555+0000] {python.py:194} INFO - Done. Returned value was: 
        CREATE TABLE air_quality (
            id serial primary key, 
            city varchar(250),
            country varchar(250), 
            unit varchar(20), 
            last_value float8, 
            lat float8,
            lng float8
        )
    
[2023-12-28T16:55:13.633+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=air_quality, task_id=Create_Table, execution_date=20231219T040000, start_date=20231228T165512, end_date=20231228T165513
[2023-12-28T16:55:14.065+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-28T16:55:14.188+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
